BitwardenShared/UI/Auth/TwoFactorNotice/EmailAccess/EmailAccessProcessor.swift

```diff
@@ -0,0 +1,76 @@
+import Combine
+import SwiftUI
+
+// MARK: - EmailAccessProcessor
+
+/// The processor used to manage state and handle actions for the new device notice screen.
```

Comment:
⛏️ Should this be like this?
```suggestion
/// The processor used to manage state and handle actions for the email access screen.
```